### DISCUSSION TITLE
The official website heading is hard to read due to color https://www.mend.io/free-developer-tools/renovate/

### DISCUSSION THREAD

**Discussion by IAFahim** [renovatebot/renovate#17527] [Suggest an Idea]
### What would you like Renovate to be able to do?

![Screenshot_20220831-023120](https://user-images.githubusercontent.com/63500913/187537990-d9d4215a-b463-46a8-85a5-e85a18f9ccf5.png)
Here you may change to color to white as on such dark blue color dark font doesn't seem hard to read

### If you have any ideas on how this should be implemented, please tell us here.

Change the font color to something else simple css stuff. It's a suggestion I could be wrong. Ow and I love this project. You guys are awesome

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
That website is not part of this open source project and cannot be addressed here. FYI it seems the blue-on-blue problem you see happens when it's <=768px
